### PR TITLE
Update ft_workflow_project.md

### PR DESCRIPTION
**Pull request by annebenoit**
Updating my project report

### CODE REVIEW

File: _projects/ft_workflow_project.md [hunk 1]
```diff
@@ -102,26 +102,42 @@ derive a clear model for replication, as there are many ways to implement it,
...
+While not meeting in person, we have stayed in close collaboration through regular Skype meetings,
+which allowed us to make progress on the project. 
+
 
 ## Impact and publications
 
-Two papers have been accepted to FTXS'17 {% cite benoitEtAl2017identifying --file external/ft_workflow_project.bib
-%},{% cite benoitEtAl2017optimal --file external/ft_workflow_project.bib
-%}.
+Two papers have been accepted to FTXS'17 {% cite benoitEtAl2017identifying --file external/ft_workflow_project.bib %},{% cite benoitEtAl2017optimal --file external/ft_workflow_project.bib %}.
```
Comment (pancetta) [inline _projects/ft_workflow_project.md:135]: Are those publications coming from this JLESC project, with partners from JLESC labs? If so, please add it to the [jlesc.bib](https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib) and cite from there.

File: _projects/ft_workflow_project.md [hunk 2]
```diff
@@ -102,26 +102,42 @@ derive a clear model for replication, as there are many ways to implement it,
 even for a single task.
 
 
-## Results for 2016/2017
+## Results
+
```
Comment (pancetta) [inline _projects/ft_workflow_project.md:106]: Could you split this to 2016/2017 and 2017/2018? I'm not sure this is needed, just asking for now..

**Review by pancetta — CHANGES_REQUESTED** [review]

File: _projects/ft_workflow_project.md (hunk 1 above)
Comment (annebenoit) [reply _projects/ft_workflow_project.md:135]: Yes they are. I'll try to do the modification.

File: _projects/ft_workflow_project.md (hunk 2 above)
Comment (annebenoit) [reply _projects/ft_workflow_project.md:106]: Yes I can do that. I will edit the file again.

**Pull request closed without merging**